src/components/base/base-link/BaseLink.scss

```diff
@@ -0,0 +1,76 @@
+@mixin createHoverState() {
+  @media (hover) {
+    &:active,
+    &:hover {
+      &:not(:disabled) {
+        @content;
+      }
+    }
+  }
+}
+
+.base-link {
+  $animationTime: 0.4s;
+
+  display: inline-flex;
+  align-items: center;
+  flex-direction: row;
+  gap: var(--spacing-40, 0);
+  color: var(--color-primary-20, #000);
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+
```

Comment:
para que aparezca el curso cuando es una a solo si tiene la property href hay que quitar el cursor: pointer desde la clase base-link y añadir este código:

&:is(a) {
    &[href] {
      cursor: pointer;
    }
  }
  

  &:is(button) {
    cursor: pointer;
  }